Solutions/Network Session Essentials/ReleaseNotes.md

```diff
@@ -1,6 +1,7 @@
 | **Version** | **Date Modified (DD-MM-YYYY)** | **Change History**                                                    |
 |-------------|--------------------------------|-----------------------------------------------------------------------|
-| 3.0.6       | 4-07-2025                     | Summary rules added and updated requiredDataConnectors.			       |
+| 3.0.8       | 23-09-2025                    | Updated analytic rule queries by lowering the threshold for event per second (eps) filtering.
```

Comment:
Removed duplicate and redundant query blocks that previously handled different eps ranges, reducing code complexity and improving maintainability

Reply:
can we add above note?